api/migrations/0004_alter_coordinator_legacy_cid_alter_group_legacy_gid_and_more.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 5.0.4 on 2024-04-28 12:34
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0003_fake_data"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="coordinator",
+            name="legacy_cid",
+            field=models.IntegerField(help_text="Coordinator id from the old database", null=True),
```

Comment:
Please, rebase your branch, remove this file and do: `make recreate-first-migration`

Reply:
I did this now